### ISSUE TITLE
"Python has stopped Working" on reaching the line "model.add(LSTM..." in the function below

### ISSUE THREAD

**Issue by shuvam26** [shuvam26/-Python-has-stopped-Working-on-reaching-the-line-model.add-LSTM...-in-the-function-below#1]
def _build_network(self, vocab_size, maxlen, 
    embedding_dimension=256,hidden_units=256, trainable=False):
        print('Build model...')
        model = Sequential()
        print('Reached here')
        model.add(Embedding(vocab_size, embedding_dimension, 
        input_length=maxlen, 
        embeddings_initializer='glorot_normal'))
        print('embedding done')
        model.add(Convolution1D(hidden_units, 3, kernel_initializer='he_normal', 
        padding='valid', activation='sigmoid',input_shape=(1, maxlen)))
        # model.add(MaxPooling1D(pool_size=3))
        model.add(Convolution1D(hidden_units, 3, kernel_initializer='he_normal', 
        padding='valid', activation='sigmoid',input_shape=(1, maxlen - 2)))
        print('conv1dcomplete')
        # model.add(MaxPooling1D(pool_size=3))
        # model.add(Dropout(0.25))
        **# "Python has stopped working" at the line below**
        model.add(LSTM(hidden_units, kernel_initializer='he_normal', 
        activation='sigmoid', dropout=0.5, return_sequences=True))
        #print('Reached here')
        model.add(LSTM(hidden_units, kernel_initializer='he_normal', 
        activation='sigmoid', dropout=0.5))
        model.add(Dense(hidden_units, kernel_initializer='he_normal', 
        activation='sigmoid'))
        model.add(Dense(2))
        model.add(Activation('softmax'))
        adam = Adam(lr=0.0001)
        model.compile(loss='categorical_crossentropy', optimizer=adam, metrics=
        ['accuracy'])
        print('No of parameter:', model.count_params())
        print(model.summary())
        return model

    Running using Anaconda-spyder Python 3.6 with latest versions of keras and 
    tensorflow.
